log/corelog/console_logger.go

```diff
@@ -63,6 +63,10 @@ func (l *consoleLogger) LogMessage(message string, fields MessageFields) {
 	}
 }
 
+func (l consoleLogger) LogEvent(interface{}, EventLogFields) {
+
+}
```

Comment:
I left a larger comment related to this line as well but I will leave just a reminder here. 

Should not this contain that actual logic instead of the higher level logger? And also maybe they could perform a more specialized feature like `logBitriseStartedEvent` then `logBitriseFinishedEvent` and so on.